Shared/Samples/Show grid/ShowGridView.swift

```diff
@@ -0,0 +1,318 @@
+        
+        /// The color of the grid lines.
+        @State private var gridColor = Color(uiColor: .red)
+        
+        /// The color of the grid labels.
+        @State private var labelsColor = Color(uiColor: .red)
+        
+        /// The kind of grid to display
+        @State private var gridType: GridType = .latLon
+        
+        /// The positioning of the grid's labels.
+        @State private var labelPosition: ArcGIS.Grid.LabelPosition = .allSides
+        
+        /// The format used for labeling the grid.
+        @State private var labelFormat: LatitudeLongitudeGrid.LabelFormat = .decimalDegrees
+        
+        /// The units used for labeling the USNG grid.
+        @State private var usngLabelUnit: USNGGrid.LabelUnit = .kilometersMeters
+        
+        /// The units used for labeling the MGRS grid.
+        @State private var mgrsLabelUnit: MGRSGrid.LabelUnit = .kilometersMeters
+        
+        var body: some View {
+            Form {
+                Section("Grid Line Settings") {
+                    Picker("Grid Type", selection: $gridType) {
+                        ForEach(GridType.allCases, id: \.self) { type in
+                            Text(type.label)
+                        }
+                    }
+                    .onChange(of: gridType) { _ in
+                        grid = gridType.makeGrid()
+                        updateUI()
+                    }
+                    
+                    Toggle("Visible", isOn: $gridIsVisible)
+                        .onChange(of: gridIsVisible) { _ in
+                            grid.isVisible = gridIsVisible
+                        }
+                    
+                    ColorPicker("Color", selection: $gridColor)
+                        .onChange(of: gridColor) { _ in
+                            changeGridColor(to: UIColor(gridColor))
+                        }
+                }
+                
+                Section("Labels Settings") {
+                    Toggle("Visible", isOn: $labelsAreVisible)
+                        .onChange(of: labelsAreVisible) { _ in
+                            grid.labelsAreVisible = labelsAreVisible
+                        }
+                    
+                    ColorPicker("Color", selection: $labelsColor)
+                        .onChange(of: labelsColor) { _ in
+                            changeLabelColor(to: UIColor(labelsColor))
+                        }
+                    
+                    Picker("Position", selection: $labelPosition) {
+                        ForEach(Grid.LabelPosition.allCases, id: \.self) { position in
+                            Text(position.label)
+                        }
+                    }
+                    .onChange(of: labelPosition) { _ in
+                        grid.labelPosition = labelPosition
+                    }
+                    
+                    if grid is LatitudeLongitudeGrid {
```

Comment:
since we are typecasting the grid later on line 140 we might as well do it here:
 (`if let latitudeLongitudeGrid = grid as? LatitudeLongitudeGrid`)